examples/compilation/library.jl

```diff
@@ -70,14 +70,18 @@ const toolchain = Ref{Toolchain}()
 function discover_toolchain()
     # Check availability NVCC
     if haskey(ENV, "NVCC")
-        nvcc = get(ENV, "NVCC")
-    else
+        nvcc = ENV["NVCC"]
+    elseif haskey(ENV, "CUDA_PATH")
```

Comment:
On windows, a default installation of CUDA will install this environment variable, so it's good to include it here for automatic detection at least.